static/src/stylesheets/module/experiments/_acquisitions-epic-testimonials.scss

```diff
@@ -0,0 +1,81 @@
+@import 'svg';
```

Comment:
Could we move all svg includes into the template and not use it as a background? This gives us the ability to reuse the existing one and work with `fill` and `stroke` without duplicating the markup.